### DISCUSSION TITLE
Sound profiles

### DISCUSSION THREAD

**Discussion by tb59427** [iltis42/XCVario#191] [Ideas]
Hi all,

the amount of settings for sound is - uhmmm - overwhelming to says the least. Is there interest to share sound profiles here? Could be a screenshot with your settings for starters. Maybe if there's enough interest, Eckhard can be convinced to make such profiles im-/exportable.

I am a newbie to xcvario so for the time being I have nothing to contribute. Sound settings close to legacy devices (e.g. lx) could be most interesting, I suppose (people get used to the sound of their varios and love to retain it, I guess).

cheers
Torsten

**Comment by iltis42** [comment]
Recent software contains a feature to save settings in a .csv settings file that stores everything different from default, so import/export is available for whole config,  and the .csv is a text file that can be edited, so sound parameter's may be merged with own config in an editor and imported via OTA management webpage (software update and more webpage).

**Reply by tb59427** [reply]
Thanks - great idea!
